pkg/tools/clockmock/mock.go

```diff
@@ -42,20 +42,48 @@ func New() *Mock {
 	}
 }
 
+// Start starts mock time to run with the given speed until ctx becomes done. While time is running, current time for
+// the mock will be the following:
+//   mock time := mock start time  +  (real time duration from the start) * speed  +  mock duration added with Set, Add
+func (m *Mock) Start(ctx context.Context, speed float64) {
+	m.lock.RLock()
+	defer m.lock.RUnlock()
+
+	const tick = 10 * time.Millisecond
```

Comment:
Should tick be configurable?

Reply:
It can, but I can get no usecase why it should.